### PR TITLE
Add ktor

### LINKED ISSUE #8611: Icon Request: Ktor
### Brand Name

Ktor

### Website

https://ktor.io/

### Popularity Metric

GitHub stars: 11.1k

See: https://github.com/ktorio/ktor

### Official Resources for Icon and Color

svg: https://ktor.io/docs/images/ktor_logo.svg
Color: `#000000`

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by f-teruhisa**
image

**Issue:** closes #8611

**Similarweb rank:** [281,332](https://www.similarweb.com/website/ktor.io/)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
`hex` taken from "Color Palette" from [Brand Assets](https://www.jetbrains.com/company/brand/)

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7990,6 +7990,12 @@
             "hex": "FF6600",
             "source": "https://ktm.com"
         },
+        {
+            "title": "ktor",
+            "hex": "087CFA",
+            "source": "https://ktor.io",
+            "guidelines": "https://www.jetbrains.com/company/brand/"
```
Comment (adamrusted) [inline _data/simple-icons.json:7997]:
```suggestion
            "source": "https://www.jetbrains.com/company/brand/#logos-and-icons-product-icons",
            "guidelines": "https://www.jetbrains.com/company/brand/#brand-guidelines"
```

File: icons/ktor.svg
Comment (adamrusted) [inline icons/ktor.svg:1]: image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for submitting a PR @f-teruhisa!

I'd pull the icon from the JetBrains brand source too, see suggested change on the JSON.
Based on your current SVG (black and white) I'm seeing some pretty major differences to the source (colour).

Would you mind having anothre look please?

**Comment by f-teruhisa** [comment]
@adamrusted 
Thank you  for your comments and suggestions. 
I'll take a look at it.

**Comment by f-teruhisa** [comment]
@adamrusted 
Thanks for reviewing! 
I fixed the image in this commit 👇  , so please review it again!

https://github.com/simple-icons/simple-icons/pull/10488/commits/67811b592d1d7a0d30258ffabf85a39987359926

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8011,6 +8011,12 @@
             "hex": "FF6600",
             "source": "https://ktm.com"
         },
+        {
+            "title": "ktor",
```
Comment (service-paradis) [inline _data/simple-icons.json:8015]: I think we should use a capital `K` here:
```suggestion
            "title": "Ktor",
```

File: icons/ktor.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ktor</title><path d="m15.979 23.982 3.983-3.984 4.017-4.016-4.224-4.224-3.776-3.776-.024 8.2zM8 16l3.983 3.983L16 24l4.224-4.224L24 16l-8.2-.024ZM8.035.018 4.052 4.001.035 8.018l4.224 4.224 3.776 3.776-.033-8.175zM16 8l-3.983-3.983L8 0 3.776 4.224 0 8l8.175-.034Z"/></svg>
```
Comment (service-paradis) [inline icons/ktor.svg:1]: When using Ktor product icons from https://www.jetbrains.com/company/brand/#logos-and-icons-product-icons, I'm seeing some differences:
image

**Review by service-paradis — CHANGES_REQUESTED** [review]

**Comment by f-teruhisa** [comment]
@service-paradis 
Thank you for reviewing.
I fixed it at this commit, so please review it.

https://github.com/simple-icons/simple-icons/pull/10488/commits/ef756d63b6c92b62bbef1aad0163e811d028b18b

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you for your perseverance on this one! There is something that went a little bit wrong at the top right corner:
image

**Comment by f-teruhisa** [comment]
@service-paradis 
Thanks Checking! 
I fixed the image, so review it again.

image

ref: https://github.com/simple-icons/simple-icons/pull/10488/commits/e706e8b9fff54bf09622bf91a434180141ca65e9

File: icons/ktor.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Ktor</title><path d="M8.001 0 8 .002 4.02 3.984l-.244.243L.002 8 0 8.002h.004l4.223 4.223L8 16l.023-7.973h.174l7.782-.025-.024 7.975h-.154l-7.8.023 3.981 3.982 3.997 3.997v.003l.002-.002L16 24l4.224-4.225L24 16h-.04l.019-.018-4.223-4.224-3.776-3.776-3.965-3.962L8.001.004Zm11.997 11.942L16 8.002v8h8z"/></svg>
```
Comment (service-paradis) [inline icons/ktor.svg:1]: Thank you for the change. Unfortunately, there is still a difference between yours (in black) and the original:
image

There are also some extra nodes that could be removed. Here is some examples, but there is more:
image
image
image

**Review by service-paradis — CHANGES_REQUESTED** [review]

**Comment by f-teruhisa** [comment]
@service-paradis 
Thank you for your checking!!
I fixed the image👇

https://github.com/simple-icons/simple-icons/pull/10488/commits/a43e635ee0a8b24c10a50aef74d9350012deebaf

And I removed extra nodes, so review again plz 🙏 

image

**Review by service-paradis — APPROVED** [review]
Perfect 🎉 Thank you @f-teruhisa!

**Pull request merged**